charts/cloudzero-prometheus-agent/templates/cm.yaml

```diff
@@ -136,7 +136,7 @@ data:
           follow_redirects: true
           enable_http2: true
     remote_write:
-      - url: 'https://{{ .Values.cloudzero.host }}/v1/container-metrics?cluster_name={{.Values.cloudzero.cluster_name}}&cloud_account_id={{.Values.cloudzero.cloud_account_id}}'
+      - url: 'https://{{ .Values.host }}/v1/container-metrics?clusterName={{.Values.clusterName}}&cloudAccountId={{.Values.cloudAccountId}}'
```

Comment:
are `clusterName` and `cloudAccountId` for the params correct? I think we'd want to keep those. we didn't change that in the query params we set in our API

Reply:
Ooops.